src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costDay = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 7) {
+    return ((costDay * days) - bigDiscount);
```

Comment:
`costDay * days` looks like a new var (e.g. `total`)
also, you don't actually need to add extra parentheses 
```suggestion
    return total - bigDiscount;
```